### PR TITLE
[ACS-6140] - Remove references to internal Angular Material CSS classes

### PR DESCRIPTION
**Pull request by DominikIwanek**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [x] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/ACS-6140

**What is the new behaviour?**
All internal Angular Material CSS classes are removed and replaced with our custom classes. Changes in Angular Material library with classes won't affect our app.
Also it contains adf-upstream

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.ts [hunk 1]
```diff
@@ -62,8 +62,7 @@ import { CategoryService } from '@alfresco/adf-content-services';
   host: { class: 'aca-rule-details' }
 })
 export class RuleDetailsUiComponent implements OnInit, OnDestroy {
-  @Input()
-  readOnly: boolean;
+  readOnly = true;
```
Comment (DenysVuika) [inline projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.ts:65]: if this property is not used, can be removed as well

File: projects/aca-content/src/lib/ui/theme.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable selector-class-pattern */
```
Comment (DominikIwanek) [inline projects/aca-content/src/lib/ui/theme.scss:1]: Do you guys think it's allowable to disable that styleline rule for theme files? I think it's really difficult to modify those files as they mainly focus on adapting mat classes

Comment (MichalKinas) [reply projects/aca-content/src/lib/ui/theme.scss:1]: I think it's okay for me

**Review by MichalKinas — APPROVED** [review]
LGTM, one final question did you run ACA with the latest ADF version to check styling that you've changed?

**Comment by DominikIwanek** [comment]
Yes, I have, 2 times:
1. while making changes, I was working on ACA and ADW with  adf configurations, to see if my changes don't break anything.
2. before creating this PR I have also walk through all of those places which have changed

And I didn't find anything suspicious :)

**Pull request merged**